services/sessions.js

```diff
@@ -40,8 +40,8 @@ var isValidPassword = function(user, password) {
 };
 
 app.get('/services/session', function(req, res) {
-	if (req.user) {
-		res.send({id: req.user.id, user: _.omit(req.user.toJSON(), "password")});
+	if (req.body.user) {
```

Comment:
It isn't. That was leftover from before I put in the filter for post and put. Removed.